### PR TITLE
fix: patching cdi after availability

### PR DESCRIPTION
**Pull request by nestoracunablanco**
This error was identified while using CodeReady Workspaces with limited resources. The DataVolume specified in the test-linux.sh file did not fully deploy, resulting in failed end-to-end tests.

**What this PR does / why we need it**: fixes error

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note NONE

```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -179,10 +179,10 @@ echo "Deploying CDI"
 oc apply -f https://github.com/kubevirt/containerized-data-importer/releases/download/$CDI_VERSION/cdi-operator.yaml
 oc apply -f https://github.com/kubevirt/containerized-data-importer/releases/download/$CDI_VERSION/cdi-cr.yaml
 
-oc patch cdi cdi -n cdi --patch '{"spec": {"config": {"dataVolumeTTLSeconds": -1}}}' --type merge
-
 oc wait --for=condition=Available --timeout=${timeout}s CDI/cdi -n cdi
 
+oc patch cdi cdi -n cdi --patch '{"spec": {"config": {"dataVolumeTTLSeconds": -1}}}' --type merge
```
Comment (ksimon1) [inline automation/test.sh:184]: What is the reason for this change?

Comment (nestoracunablanco) [reply automation/test.sh:184]: @ksimon1 I'm using CodeReady Containers for development on my workstation. While I'm not entirely sure of the underlying reasons, some empirical testing has shown that this change allows the tests to work. Otherwise, they fail when applying the data volumes, as the persistent volume claim is unable to claim any storage.

Additionally, I've encountered the same issue when installing an OpenShift cluster in a VM and running it with QEMU. Do you have any insights into why this might be happening?

**Comment by nestoracunablanco** [comment]
@ksimon1 @fossedihelm, do you want any changes to the code? is additional clarification needed for this PR to be merged?

**Review by 0xFelix — APPROVED** [review]
/lgtm
